pmultiqc/modules/quantms/proteobench.py

```
@@ -209,84 +275,71 @@
-
-    data_range_min = data_list.min() - (data_list.min() * 0.05)
-
     bin_step = 1000
-
-    data_bins = np.arange(data_range_min, data_range_max + (data_range_max * 0.05), (data_range_max / bin_step))
+    data_bins = np.arange(
+        data_range_min,
+        data_range_max + (data_range_max * 0.05),
+        (data_range_max / bin_step)
+    )
     bin_mid = (data_bins[: -1] + data_bins[1: ]) / 2
```

Flake8 (7.2.0)

[error] whitespace before ']'

(E202)